tslint.json

```diff
@@ -1,21 +1,14 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-        "tslint:recommended"
-    ],
-    "jsRules": {},
-    "rules": {
-        "interface-name": [ "never-prefix" ],
-        "member-access": [
-            false
-        ],
-        "ordered-imports": [
-            false
-        ],
-        "no-console": [
-            false
-        ],
-        "no-var-requires": false
-    },
-    "rulesDirectory": []
-}
\ No newline at end of file
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended"],
+  "jsRules": {},
+  "rules": {
+    "interface-name": ["never-prefix"],
+    "member-access": [false],
+    "ordered-imports": [false],
+    "no-console": [false],
+    "no-var-requires": false,
+    "trailing-comma": ["never"]
```

Comment:
Prettier and ts were arguing over trailing commas so set this to never. Otherwise file is the same.